packages/backend/src/app.ts

```diff
@@ -553,8 +553,6 @@ export class App {
       quoteRoutes.get
     )
 
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This was the one specifically called out in the original issue https://github.com/interledger/rafiki/issues/917 and I could use a sanity check on the changes. Here is the full commit: https://github.com/interledger/rafiki/pull/2340/commits/7237c7090cf46970b897a6fb1bb079b5d058c83f

Mostly want to double check that the type and tests are safe to remove. I believe it should have been removed alongside the other connection stuff when we replaced it with incoming payment methods. I only see one place the spsp middleware is used (here) so the removed context type and tests should no longer be needed.